pycontrails/models/cocip/cocip.py

```diff
@@ -2372,6 +2383,12 @@ def calc_timestep_contrail_evolution(
     contrail_2["ef"] = energy_forcing_2
     contrail_2["age"][energy_forcing_2 == 0.0] = np.timedelta64(0, "ns")
 
+    if params["compute_atr20"]:
+        contrail_2["global_mean_rf"] = (
+            contrail_2["ef"] / constants.surface_area_earth / constants.seconds_per_year
+        )
+        contrail_2["atr20"] = params["global_rf_to_atr20_factor"] * contrail_2["global_mean_rf"]
```

Comment:
It's not obvious to me why a year is the right length of time to use in the conversion from EF to RF. Is it just because the conversion factor (`params["global_rf_to_atr20_factor"]`) is defined for contrail RF averaged over a year?

If that's right, then I think we should change `"global_mean_rf"` to `"global_yearly_mean_rf"`, or something similar, to make it clearer that this is a quantity that depends on *both* the area and time period used for normalization.

Reply:
Yes, i should probably explicitly reference equation (S5) of the supplement to paper: https://gmd.copernicus.org/articles/16/3313/2023/gmd-16-3313-2023-discussion.html